src/lsp/cobol_ptree/terms_visitor.ml

```diff
@@ -232,9 +250,42 @@ and fold_address (v: _ #folder) =
 
 and fold_inline_call (v: _ #folder) =
   handle v#fold_inline_call
-    ~continue:begin fun { call_fun; call_args } x -> x
-      >> fold_name' v call_fun
-      >> fold_list ~fold:fold_effective_arg v call_args
+    ~continue:begin function
```

Comment:
In this case you should `begin fun c x -> match c with` and then `<pattern> -> x` for clarity.